minio/minio
# Remove redundant namespace from the helm chart

## Description
The PR removes redundant `namespace` from the chart templates.

## Motivation and Context
Motivation - keep code smaller and cleaner.
Context - Helm creates Kubernetes resources (Deployment, ConfigMap, Service, Ingress etc.) in the same namespace where release installed, i.e. `.Release.Namespace` is used by default.

## How to test this PR?
Install the chart to `qwerty` namespace:
```shell
$ helm install test helm/minio -n qwerty
NAME: test
LAST DEPLOYED: Tue Apr  4 00:46:11 2023
NAMESPACE: qwerty
STATUS: deployed
REVISION: 1
TEST SUITE: None
NOTES:
   ...
```
Check release installed to `qwerty` namespace:
```shell
$ helm list --all-namespaces
NAME   NAMESPACE   REVISION  UPDATED                                STATUS     CHART        APP VERSION
test   qwerty      1         2023-04-04 00:46:11.2516625 +0200 CEST deployed   minio-5.0.7  RELEASE.2023-02-10T18-48-39Z
```
List Kubernetes resources in  `qwerty` namespace:
```shell
$ kubectl get all -n qwerty
NAME               READY   STATUS    RESTARTS   AGE
pod/test-minio-0   1/1     Running   0          34s
pod/test-minio-1   1/1     Running   0          34s
pod/test-minio-2   1/1     Running   0          34s
pod/test-minio-3   1/1     Running   0          34s

NAME                         TYPE        CLUSTER-IP      EXTERNAL-IP   PORT(S)    AGE
service/test-minio           ClusterIP   10.107.11.200   <none>        9000/TCP   34s
service/test-minio-console   ClusterIP   10.111.49.174   <none>        9001/TCP   34s
service/test-minio-svc       ClusterIP   None            <none>        9000/TCP   34s

NAME                          READY   AGE
statefulset.apps/test-minio   4/4     34s
```

## Types of changes

## Checklist:
